client/src/featureform/register.py

```diff
@@ -3662,6 +3696,26 @@ def avg_user_transactions(client, params, entities):
             return decorator
         else:
             return decorator(fn)
+        
+
+    def register_trigger(self, trigger_name, trigger_type):
```

Comment:
Should just have this `ff.ScheduleTrigger(name=”Monday at Noon”, "0 12 * * MON")` that registers the trigger